pkg/console/operator/status.go

```diff
@@ -84,6 +84,9 @@ func (c *consoleOperator) HandleAvailable(operatorConfig *operatorsv1.Console, p
 
 // internal func for handling conditions
 func handleCondition(operatorConfig *operatorsv1.Console, conditionType string, reason string, err error) {
+	if conditionType == "" {
```

Comment:
I think a `panic` here is probably correct, we shouldn't allow empty string.

Reply:
Wont this cause program to crash unless `recover()` func is called ?

Reply:
It's really for development.  We shouldn't do this, so it should crash.  Not sure if there is a better way to ensure this?

Reply:
eh, i will eliminate.  its obvious "dont do that"